scripts/release.sh

```diff
@@ -631,6 +636,7 @@ ALL_IN_ONE="true"
 INCREMENTAL="false"
 DEFAULT_BINARIES=${BUILD_BINARIES:-}
 BUILD_BINARIES=
+STATIC_LINKING=true
```

Comment:
All in one is using quotes, to add it here or remove there I guess

```suggestion
STATIC_LINKING="true"
```